packages/vuetify/src/components/VDatePicker/VDatePicker.ts

```diff
@@ -65,6 +65,10 @@ export default mixins(
       type: [String, Number],
       default: 0,
     },
+    firstDayOfYear: {
```

Comment:
I don't quite get the idea of this prop... Why first day of the year is thursday by default? When it could be/should be changed? What's the practical example of usage?

Reply:
I added the prefix locale to indicate it is a locale determined property. In the US first day of week  is Sunday and the 1st of January is always in week 1. With the ISO 8601 standard the first day of the week is Monday and the week containing 4th of January determines week 1. So localeFirstDayOfYear should be 4 then.